src/stories/Components/Tag/Tag.mdx

```diff
@@ -0,0 +1,125 @@
+import { Meta, Story, Canvas, Controls } from '@storybook/blocks'
+import * as TagStories from './Tag.stories'
+
+import { createAlert } from '../Alert/Alert.js'
+
+<Meta of={TagStories} />
+
+# Tag
+
```

Comment:
Adding a description of what elements this is intended to be used on can be helpful.

Reply:
Also clarifying that Tags are intended to be used for interaction and input.

Something like

```
"The Tag component is similar to the Badge component, however it has a different semantic purpose. Tag is intended to be used for interaction and input where Badge is intended to be used for Notification and Information. See [Badge](?path=/docs/components-badge--docs) for details on its usage."
```

And then in Badge we can update it to look like

```
"The Badge component is similar to the Tag component, however it has a different semantic purpose. Badge is intended to be used for notification and information where Tag is intended to be used for interaction and input. See [Tag](?path=/docs/components-tag--docs) for details on its usage."
```